setup.py

```diff
@@ -66,7 +66,9 @@ def get_setup_version(root, reponame):
         'sphinx',
         'beautifulsoup4',
         'jinja2 <3.1',
-        'pillow'
+        'pillow',
+        'pydata-sphinx-theme <0.9.0',
```

Comment:
To expand on that, right now each project either does or does not define it's own template overrides. These are tied to the pydata-sphinx-theme version which means pinning here it will break at least some of our projects.

Reply:
Reverting that!